lib/reqTemplate.js

```diff
@@ -22,6 +22,8 @@ var compilerOptions = {
 };
 
 function compileExpression(expression, part) {
+    // Convert the expression to a single line & remove whitespace.
+    expression = expression.replace(/\n/g, ' ').trim();
```

Comment:
Wouldn't it be better to remove extra spaces after the new line as well? Also, for Windows compatibility, this should probably be `replace(/[\n\r]+\s+/g, ' ')`